subsys/bluetooth/mesh/time_srv.c

```diff
@@ -243,6 +265,11 @@ static int handle_time_set(struct bt_mesh_model *model, struct bt_mesh_msg_ctx *
 		srv->time_update_cb(srv, ctx, BT_MESH_TIME_SRV_SET_UPDATE);
 	}
 
+	/* The time server shouldn't send out statuses more often than every 30 seconds. */
+	if ((srv->data.timestamp + RESTRICTION_TIME) < srv->data.sync.uptime) {
+		(void)bt_mesh_time_srv_time_status_send(srv, NULL);
```

Comment:
It will be better to call 'send_time_status()' here, since, server shall always send response while handling Time_Set message.

Reply:
Server always sends response. Please take a look at line 273: https://github.com/nrfconnect/sdk-nrf/pull/11997/files#diff-626675688a86ce1b261757ffd4f251e64c8b334403c2432102a99ec505fa8c32R273

This is restriction for publishing Time Status as result of state change.
Please read errata:
> A Time Server should publish a Time Status message **when the TAI Seconds, the Subsecond, or the Time Zone 
> Offset Current fields of the Time state of the Time Server (see Section 5.1.1) have been updated as a result 
> of processing a Time Set message** (see Section 5.2.1.2) or a Time Status message (see Section 5.2.1.3), and 
> **the value of the Time Role state (see Section 5.1.2) is 0x01 (Time Authority) or 0x02 (Time Relay)**, 
> or the Time Server obtains a new Time state (see Section 5.1.1) from a trusted OOB source (such as a GPS or a 
> mobile phone synchronized with a cellular network).
> **The two consecutive publications of the Time Status message should be separated by at least 30 seconds.** 

Seems this code does exactly what errata require. Restricts publishing on state change over Time set for Time Relay and Time Authority.